frontend/src/metabase/search/utils/enabled-search-type/enabled-search-type.ts

```diff
@@ -0,0 +1,10 @@
+import type { EnabledSearchModelType } from "metabase-types/api";
+import { enabledSearchTypes } from "metabase/search/constants";
+
+export function isEnabledSearchModelType(
+  value: any,
```

Comment:
https://stackoverflow.com/questions/51439843/unknown-vs-any


```suggestion
  value: unknown,
```